packages/standard-validator/src/index.ts

```diff
@@ -1,6 +1,6 @@
+import type { StandardSchemaV1 } from '@standard-schema/spec'
```

Comment:
These changes are already existing in honojs:main, will have to recreate branch to fix this.